config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-
+  resources :vouchers, only: [:index, :show, :create, :filter]
```

Comment:
this is a new routes that are now working (only accessible by logged in retailer)

GET    `/vouchers`     ->    vouchers#index
POST  ` /vouchers`(      ->  vouchers#create
GET    `/vouchers/:id`   ->  vouchers#show